common/components/filter/_filter.scss

```diff
@@ -0,0 +1,67 @@
+.app-filter {
+  margin: 0 0 govuk-spacing(3);
+}
+
+.app-filter__list {
+  @include govuk-clearfix;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.app-filter__list-item {
+  flex-grow: 1;
+  min-height: 70px;
+  display: flex;
+}
+
+.app-filter__list-item--active {
+  background: govuk-colour('white');
+  border: 2px solid govuk-colour('black');
+  padding: govuk-spacing(4);
+  padding-right: govuk-spacing(6);
+  @include govuk-font($size: 19, $weight: normal, $line-height: 1.25);
+}
+
+.app-filter__link {
+  // cannot remove text-decoration from parts of the link, so we remove it everywhere and re-add it when necessary
+  text-decoration: none;
+  background: govuk-colour('blue');
+  border: 2px solid govuk-colour('blue');
+  color: govuk-colour('white');
+  display: block;
+  padding: govuk-spacing(4);
+  padding-right: govuk-spacing(6);
+  flex-grow: 1;
+
+  @include govuk-font($size: 19, $weight: normal, $line-height: 1.25);
+
+  &:link,
+  &:active,
+  &:visited {
+    color: govuk-colour('white');
+  }
+
+  &:hover {
+    color: #d5e8f3;
+  }
+
+  &:focus {
+    background-color: govuk-colour('blue');
+  }
+
+  @include govuk-media-query($media-type: print) {
```

Comment:
Might be worth having a look at the mobile view too:

![image](https://user-images.githubusercontent.com/3327997/76511864-a57d5280-644b-11ea-890f-b1f6dcd7a621.png)

We could stack them, or maybe change the layout.